web/src/components/UploadAirgapBundle.tsx

```diff
@@ -64,16 +60,21 @@ type State = {
   uploadResuming: boolean;
   viewOnlineInstallErrorMessage: boolean;
 };
+type BundleFile = {
+    name: string;
+  } | null;
 const UploadAirgapBundle = (props: Props) => {
+  // TODO: remove refs and use state instead
+  const bundleFile = useRef<BundleFile>(null);
+  const errorMessage = useRef<string>("");
```

Comment:
Not sure how to fix this the "right" way but using refs is a pretty easy shortcut. Added a todo to refactor this later.

Reply:
Cool! I'm forgettign but do we have it documented somewhere of why errorMessage became a stale state?